### PR TITLE
DPC-1245 Add SSAS to V2 stack

### PR DESCRIPTION
**Pull request by MrMorie**
### Fixes [DPC-1245](https://jira.cms.gov/browse/DPC-1245)

For DPC v2, we need a Dockerfile that will run SSAS.
 * ACO-API has [just such a file](https://github.com/CMSgov/bcda-app/blob/master/Dockerfiles/Dockerfile.ssas) already
 * We should include this in the Makefile `start-v2`and `down-v2`
 
### Proposed Changes

- add ssas docker file
- add ssas container to v2 network/stack

### Change Details

- Added SSAS to make down-v2
- Added SSAS to make start-v2
- Added SSAS to make build-v2
- Added make load-ssas-fixtures

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [x] no PHI/PII is affected by this change

### Acceptance Validation

Screen Shot 2021-06-21 at 6 04 29 PM

### CODE REVIEW

File: tools.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+package main
+
+import (
+    // bcda/worker/ssas dependencies
+    _ "github.com/BurntSushi/toml"
+    _ "github.com/go-delve/delve/cmd/dlv"
+    _ "github.com/howeyc/fsnotify"
+    _ "github.com/mattn/go-colorable"
+)
```
Comment (dhgreene) [inline tools.go:11]: How will we be using this? Do we need another entry in the `Makefile`?

File: docker-compose.v2.yml [hunk 2]
```diff
@@ -39,6 +39,58 @@ services:
...
+      - OKTA_CLIENT_ORGURL=https://cms-sandbox.oktapreview.com
+      - OKTA_EMAIL=shawn@bcda.aco-group.us
+      - OKTA_CLIENT_TOKEN=${OKTA_CLIENT_TOKEN}
+      - BCDA_AUTH_PROVIDER=${BCDA_AUTH_PROVIDER}
+      - OKTA_OAUTH_SERVER_ID=${OKTA_OAUTH_SERVER_ID}
+      - OKTA_CA_CERT_FINGERPRINT=${OKTA_CA_CERT_FINGERPRINT}
+      - OKTA_MFA_EMAIL=${OKTA_MFA_EMAIL}
+      - OKTA_MFA_USER_ID=${OKTA_MFA_USER_ID}
+      - OKTA_MFA_USER_PASSWORD=${OKTA_MFA_USER_PASSWORD}
+      - OKTA_MFA_SMS_FACTOR_ID=${OKTA_MFA_SMS_FACTOR_ID}
```
Comment (dhgreene) [inline docker-compose.v2.yml:64]: These (and the `OKTA_*` lines above) are not required, and DPC won't be using them.

File: tools.go (hunk 1 above)
Comment (MrMorie) [reply tools.go:11]: it was required by the ssas Dockerfile

File: docker-compose.v2.yml (hunk 2 above)
Comment (MrMorie) [reply docker-compose.v2.yml]: removed

**Review by dhgreene — APPROVED** [review]
🙌

**Pull request merged**